zoomerang.js

```diff
@@ -4,6 +4,18 @@
 
 (function () {
 
+    // Prefix helper
```

Comment:
the only prefix needed for zoom-in and zoom-out cursors is `-webkit-`: http://caniuse.com/#feat=css3-cursors-newer